addons/event/tests/test_event_security.py

```diff
@@ -86,8 +86,8 @@ def test_event_access_event_manager(self):
         registration = self.env['event.registration'].create({'event_id': event.id, 'name': 'Myself'})
         registration.write({'name': 'Myself2'})
 
-        stage = self.env.ref('event.event_stage_done')
-        stage.write({'name': 'ManagerEnded'})
+        stage = self.env.ref('event.event_stage_cancelled')
```

Comment:
Why this change ?

Reply:
The problem was that, due to the fact that I had to add an event in the "done" stage, this test would break when trying to unlink the stage as it would still be referenced by said event :/. 

To prevent this, a cleaner solution would be to create a stage in the test and delete it to avoid relying on data and stumble on the same kind of problem further down the line.